java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -91,4 +89,118 @@ protected static ConnectionManager buildConnectionManager(ChannelHandler channel
     protected static CommandManager buildCommandManager(ChannelHandler channelHandler) {
         return new CommandManager(channelHandler);
     }
+
+    /**
+     * Extracts the response from the Protobuf response and either throws an exception or returns the
+     * appropriate response as an <code>Object</code>.
+     *
+     * @param response Redis protobuf message
+     * @return Response <code>Object</code>
+     */
+    protected Object handleObjectResponse(Response response) {
+        // convert protobuf response into Object
+        return new BaseCommandResponseResolver(RedisValueResolver::valueFromPointer).apply(response);
+    }
+
+    /**
+     * Checks that the Response is empty.
+     *
+     * @return An empty response
```

Comment:
May be update the doc?
```suggestion
     * Checks that the Response is OK.
     *
     * @return An OK response
```